src/components/Map.svelte

```diff
@@ -43,6 +43,7 @@
       mapStyle: map,
       numberOfMaps: num,
     };
+    setMapComponent(mapRenderer);
```

Comment:
Had to add this line for consistent prop being passed, since `setProps` gets called again and was making `mapRenderer` undefined.